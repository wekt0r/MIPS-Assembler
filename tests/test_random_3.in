mfhi $s6
and $13, $s1, $23
add $t8, $t9, $t4
addi $9, $sp, 26943
nor $9, $28, $10
or $s1, $28, $a0
add $13, $31, $18
andi $s1, $sp, 18661
andi $13, $10, 19682
sll $zero, $t2, 12
div $29, $16
xori $t5, $10, 11931
addiu $8, $ra, 11802
lui $s3, 8489
srl $26, $s3, 23
sll $28, $4, 2
multu $5, $s6
sub $24, $1, $t0
sub $t7, $12, $k0
sub $20, $10, $27
sltu $s6, $31, $s3
mult $t5, $9
xor $t2, $s6, $fp
mfhi $9
sub $s2, $26, $t6
mult $s0, $v1
multu $t1, $7
sllv $t3, $20, $at
srav $t1, $24, $9
mfhi $28
mult $24, $9
sll $24, $4, 28
mult $26, $30
slti $29, $t0, 5027
ori $4, $29, 26961
multu $s6, $v0
srlv $3, $t3, $s0
mflo $4
nor $22, $t2, $a1
sra $31, $16, 13
xor $27, $t3, $3
srl $24, $26, 2
sltu $21, $fp, $2
divu $fp, $t4
divu $5, $28
subu $5, $s5, $k1
addu $t7, $15, $7
addiu $a0, $0, 9743
xori $24, $9, 12426
or $4, $12, $s6
subu $fp, $11, $t3
andi $23, $2, 274
sltiu $3, $4, 15952
sltu $zero, $v0, $t8
div $s7, $t5
slti $t8, $27, 16150
xori $v0, $k1, 2966
ori $0, $31, 9723
