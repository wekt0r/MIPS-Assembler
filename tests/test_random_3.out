.text
00000000	0000B010	mfhi	$s6
00000004	02376824	and	$13,$s1,$23
00000008	032CC020	add	$t8,$t9,$t4
0000000c	23A9693F	addi	$9,$sp,26943
00000010	038A4827	nor	$9,$28,$10
00000014	03848825	or	$s1,$28,$a0
00000018	03F26820	add	$13,$31,$18
0000001c	33B148E5	andi	$s1,$sp,18661
00000020	314D4CE2	andi	$13,$10,19682
00000024	000A0300	sll	$zero,$t2,12
00000028	03B0001A	div	$29,$16
0000002c	394D2E9B	xori	$t5,$10,11931
00000030	27E82E1A	addiu	$8,$ra,11802
00000034	3C132129	lui	$s3,8489
00000038	0013D5C2	srl	$26,$s3,23
0000003c	0004E080	sll	$28,$4,2
00000040	00B60019	multu	$5,$s6
00000044	0028C022	sub	$24,$1,$t0
00000048	019A7822	sub	$t7,$12,$k0
0000004c	015BA022	sub	$20,$10,$27
00000050	03F3B02B	sltu	$s6,$31,$s3
00000054	01A90018	mult	$t5,$9
00000058	02DE5026	xor	$t2,$s6,$fp
0000005c	00004810	mfhi	$9
00000060	034E9022	sub	$s2,$26,$t6
00000064	02030018	mult	$s0,$v1
00000068	01270019	multu	$t1,$7
0000006c	00345804	sllv	$t3,$20,$at
00000070	01384807	srav	$t1,$24,$9
00000074	0000E010	mfhi	$28
00000078	03090018	mult	$24,$9
0000007c	0004C700	sll	$24,$4,28
00000080	035E0018	mult	$26,$30
00000084	291D13A3	slti	$29,$t0,5027
00000088	37A46951	ori	$4,$29,26961
0000008c	02C20019	multu	$s6,$v0
00000090	020B1806	srlv	$3,$t3,$s0
00000094	00002012	mflo	$4
00000098	0145B027	nor	$22,$t2,$a1
0000009c	0010FB43	sra	$31,$16,13
000000a0	0163D826	xor	$27,$t3,$3
000000a4	001AC082	srl	$24,$26,2
000000a8	03C2A82B	sltu	$21,$fp,$2
000000ac	03CC001B	divu	$fp,$t4
000000b0	00BC001B	divu	$5,$28
000000b4	02BB2823	subu	$5,$s5,$k1
000000b8	01E77821	addu	$t7,$15,$7
000000bc	2404260F	addiu	$a0,$0,9743
000000c0	3938308A	xori	$24,$9,12426
000000c4	01962025	or	$4,$12,$s6
000000c8	016BF023	subu	$fp,$11,$t3
000000cc	30570112	andi	$23,$2,274
000000d0	2C833E50	sltiu	$3,$4,15952
000000d4	0058002B	sltu	$zero,$v0,$t8
000000d8	02ED001A	div	$s7,$t5
000000dc	2B783F16	slti	$t8,$27,16150
000000e0	3B620B96	xori	$v0,$k1,2966
000000e4	37E025FB	ori	$0,$31,9723
